proc_Hahn_echoph.py

```diff
@@ -1,100 +1,72 @@
-for searchstr, exp_type, nodename, postproc, label_str, slice_f in [
-    (
-        "200302_alex_probe_water",
-        "test_equip",
-        "signal",
-        "spincore_Hahn_echoph_v1",
-        "microwaves off",
-        (-2.5e3, 2.5e3),
-    ),
-]:
-    # {{{loads raw data and plots
-    s = find_file(
-        searchstr,
-        exp_type=exp_type,
-        expno=nodename,
-        postproc=postproc,
-        lookup=lookup_table,
-        fl=fl,
-    )
-    s.mean("nScans")
-    # }}}
-    # {{{rough centering of sliced data
-    s = s["t2":slice_f]
-    s.ift("t2")
-    rough_center = (
-        abs(s).convolve("t2", 0.01).mean_all_but("t2").argmax("t2").item()
-    )
-    s.setaxis(t2 - rough_center)
-    logger.debug(strm(ndshape(s)))
-    # }}}
-    # {{{ apply phase corrections
-    best_shift = hermitian_function_test(select_pathway(s, signal_pathway))
-    logger.info(strm("best shift is", best_shift))
-    s_uncorrected = s.C.ft("t2")
-    s.setaxis("t2", lambda x: x - best_shift).register_axis(
-        {"t2": 0}, nearest=False
-    )
-    ph0 = s["t2":0]["ph2", 0]["ph1", 1]
-    logger.info(strm(ndshape(ph0)))
-    if len(ph0.dimlabels) > 0:
-        assert len(ph0.dimlabels) == 1, (
-            repr(ndshape(ph0.dimlabels)) + " has too many dimensions"
+with psd.figlist_var() as fl:
+    for searchstr, exp_type, nodename, postproc, label_str, slice_f in [
+        (
+            "200302_alex_probe_water",
+            "ODNP_NMR_comp/ODNP/old/2020",
+            "signal",
+            "proc_Hahn_echoph",
+            "microwaves off",
+            (-2.5e3, 2.5e3),
+        ),
+    ]:
+        # {{{loads raw data and plots
+        s = psd.find_file(
+            searchstr,
+            exp_type=exp_type,
+            expno=nodename,
+            postproc=postproc,
+            lookup=prscr.lookup_table,
+            fl=fl,
         )
-        ph0 = zeroth_order_ph(ph0, fl=fl)
-        logger.info(strm("phasing dimension as one"))
-    else:
-        logger.info(
-            strm(
-                "there is only one dimension left -- standard 1D zeroth order phasing"
+        # }}}
+        # {{{rough centering of sliced data
+        s = s["t2":slice_f]
+        s.ift("t2")
+        psd.logger.debug(psd.strm(psd.ndshape(s)))
+        # }}}
+        s_uncorrected = s.C.ft("t2")
+        ph0 = s["t2":0]["ph2", 0]["ph1", 1]
+        psd.logger.info(psd.strm(psd.ndshape(ph0)))
+        if len(ph0.dimlabels) > 0:
+            assert len(ph0.dimlabels) == 1, (
+                repr(psd.ndshape(ph0.dimlabels)) + " has too many dimensions"
+            )
+            ph0 = prscr.zeroth_order_ph(ph0, fl=fl)
+            psd.logger.info(psd.strm("phasing dimension as one"))
+        else:
+            psd.logger.info(
+                psd.strm(
+                    "there is only one dimension left -- standard 1D zeroth order phasing"
+                )
             )
+            ph0 = ph0 / abs(ph0)
+        s /= ph0
```

Comment:
taken from original script - trying to make the least changes as possible from og

Reply:
I don't understand why you are modifying this script.

Reply:
see [here](https://jmfrancklab.slack.com/archives/CV553TPRV/p1724856423822879?thread_ts=1724855644.907539&cid=CV553TPRV)